src/components/character-avatar/styles.ts

```diff
@@ -0,0 +1,46 @@
+import { type MatchStage } from "~/types";
+import { theme } from "~/styles/theme";
+
+export const styles = {
+  avatar: (stage?: MatchStage, isSelected?: boolean, isBot?: boolean) => {
+    let fill = "transparent";
+
+    if (stage === "voting") {
+      fill = isSelected ? theme.palette.common.white : "transparent";
+    } else if (stage === "results") {
+      if (isSelected === false && isBot === false) {
+        fill = "transparent";
+      } else if (isSelected === true && isBot === false) {
+        fill = theme.palette.error.main;
+      } else if (isSelected === true && isBot === true) {
```

Comment:
`=== true` and `=== false` are redundant. just type it this way `isSelected || !isBot`

Reply:
etc.